### PR TITLE
Properly capitalize Brands

### PR DESCRIPTION
**Pull request by aslafy-z**
Properly capitalize **S**end**G**rid and **G**it**H**ub

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-python?pullRequest=681) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-python?pullRequest=681) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-python?pullRequest=681) it.</sub>

**Comment by thinkingserious** [comment]
Hello @aslafy-z,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/hacktoberfest-2018-has-arrived/">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/hacktoberfest/join">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by thinkingserious** [comment]
Hello @aslafy-z,

Could you please merge this against our [v4 branch](https://github.com/sendgrid/sendgrid-python/tree/v4)? Thank you!

With Best Regards,

Elmer

**Comment by aslafy-z** [comment]
@thinkingserious better now?

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -21,4 +21,4 @@ Closes #2
 - 
 - 
 
-If you have questions, please send an email to [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.
+If you have questions, please send an email to [SendGrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.
```
Comment (42B) [inline .github/PULL_REQUEST_TEMPLATE]: What about GitHub?

**Comment by thinkingserious** [comment]
Hello @aslafy-z,<br /><br/>Thanks again for the PR!<br /><br />                                         You have earned 9 out of the 5 points needed to receive glorious SendGrid Hacktoberfest swag.<br /><br />                                         Please take a moment to checkout <a href="https://dx.sendgrid.com/hacktoberfest/issues">this link</a> to find more issues to get you past the required threshold or to simply continue the celebration.<br /><br />                                         Also, <strong>please be sure you have officially registered with us <a href="https://dx.sendgrid.com/hacktoberfest/join">here</a> by November 1, 2018</strong> to qualify.<br /><br />                                         If you have any questions you can email us at <a href="mailto:dx+hacktoberfest2018@sendgrid.com">dx+hacktoberfest2018@sendgrid.com</a>.<br /><br />                                         Thank you and Happy Hacktobering!<br /><br />                                         Team SendGrid

**Pull request closed without merging**